disk/disk.go

```diff
@@ -42,6 +42,7 @@ type IOCountersStat struct {
 	WeightedIO       uint64 `json:"weightedIO"`
 	Name             string `json:"name"`
 	SerialNumber     string `json:"serialNumber"`
+	Label            string `json:"label"`
```

Comment:
I still have not know `label` is a good name or not. But other platform has some `label` even if the actual meaning is different on each platform.  Could you add some document on README at least Linux?

Reply:
@shirou I added it to the readme, thanks!